restdocs-api-spec-openapi3-generator/src/test/kotlin/com/epages/restdocs/apispec/openapi3/OpenApi3GeneratorTest.kt

```diff
@@ -730,6 +778,30 @@ class OpenApi3GeneratorTest {
         )
     }
 
+    private fun getProductRequestAsFormData(method: HTTPMethod, getSecurityRequirement: () -> SecurityRequirements = ::getOAuth2SecurityRequirement): RequestModel {
+        return RequestModel(
+                path = "/products/{id}-{subId}",
```

Comment:
This is probably due to copy&paste from another given method, but does just distract here. I'd simplify it to:
`path = "/products/{id}",`